float32 bullet_velocity  # 子弹速度
float32 bullet_angle    # 子弹角度
int16 gimbal_yaw       # 云台当前偏航角
uint8 color            # 敌方颜色(0:红色, 1:蓝色)